README.md

```diff
@@ -722,3 +744,29 @@ Validation allows only one of these three fields to be present.
     type: object
 ```
 
+### Cancel Restore Job
+
+New endpoint:  
+[`POST {{baseUrl}}/v1/restore/cancel/:<jobId>`](https://aerospike.github.io/aerospike-backup-service/#/Restore/cancel)
+
+Cancel the restore job identified by `<jobId>`. Data that has already been restored will remain intact.
+
+### Cancel Backup Job
+
+New endpoint:  
+[`POST {{baseUrl}}/v1/backups/cancel/:<routineName>`](https://aerospike.github.io/aerospike-backup-service/#/Backup/cancel)
+
+Cancel all currently running backups (both full and incremental) for the specified routine. Partially created backups will be deleted.
+
+
+### Disable Routine
+
+New endpoints:
+
+- [`POST {{baseUrl}}/v1/routines/:<routineName>/disable/`](https://aerospike.github.io/aerospike-backup-service/#/Routine/disable)
+- [`POST {{baseUrl}}/v1/routines/:<routineName>/enable/`](https://aerospike.github.io/aerospike-backup-service/#/Routine/enable)
+
+Set the disabled flag for the given routine to `true` or `false` (default is `false`).
```

Comment:
These github.io links don't go to the correct section on the live page. Can you update them, or will the openAPI spec get updated to reflect this?

Reply:
Yes, the API is not published yet.